### PR TITLE
[Renaming] Add RenameAttributeRector

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: rules/Renaming/Rector/Class_/RenameAttributeRector.php [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    {
+        $hasChanged = false;
+
+        foreach ($node->attrGroups as $attrGroup) {
+            foreach ($attrGroup->attrs as $attr) {
+                foreach ($this->renameAttributes as $renameAttribute) {
+                    if ($this->isName($attr->name, $renameAttribute->getOldAttribute())) {
+                        $attr->name = new FullyQualified($renameAttribute->getNewAttribute());
+
+                        $hasChanged = true;
```
Comment (samsonasik) [inline rules/Renaming/Rector/Class_/RenameAttributeRector.php:78]: Here needs `continue 2`, if already renamed, continue to next attr

File: rules/Renaming/Rector/Class_/RenameAttributeRector.php [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+CODE_SAMPLE
+                    ,
+                    <<<'CODE_SAMPLE'
+#[BasicRoute()]
+class SomeClass
+{
+}
+CODE_SAMPLE
+                    ,
+                    []
```
Comment (samsonasik) [inline rules/Renaming/Rector/Class_/RenameAttributeRector.php:51]: configurable code example should have entry example

File: rules/Renaming/Rector/Class_/RenameAttributeRector.php (hunk 1 above)
Comment (TomasVotruba) [reply rules/Renaming/Rector/Class_/RenameAttributeRector.php:78]: Got it, thanks :+1:

File: rules/Renaming/Rector/Class_/RenameAttributeRector.php (hunk 2 above)
Comment (TomasVotruba) [reply rules/Renaming/Rector/Class_/RenameAttributeRector.php:51]: Thanks :+1:  Fixed in next commit

**Pull request merged**